zeebe/gateway-rest/src/main/java/io/camunda/zeebe/gateway/rest/ResponseMapper.java

```diff
@@ -56,6 +60,24 @@ private static ActivatedJob toActivatedJob(final long jobKey, final JobRecord jo
         .tenantId(job.getTenantId());
   }
 
+  public static CamundaUserResponse toCamundaUserResponse(final CamundaUser camundaUser) {
+    final CamundaUserResponse camundaUserDto = new CamundaUserResponse();
+    camundaUserDto.setId(camundaUser.getId());
+    camundaUserDto.setUsername(camundaUser.getUsername());
+    camundaUserDto.setName(camundaUser.getName());
+    camundaUserDto.setEmail(camundaUser.getEmail());
+    camundaUserDto.setEnabled(camundaUser.isEnabled());
+
+    return camundaUserDto;
+  }
+
+  public static CamundaGroupResponse toGroupResponse(final CamundaGroup group) {
+    final CamundaGroupResponse camundaGroupResponse = new CamundaGroupResponse();
+    camundaGroupResponse.setId(group.id());
+    camundaGroupResponse.setName(group.name());
+    return camundaGroupResponse;
+  }
```

Comment:
🙃 Just a nitpick: let's align the names. Either both are `toCamunda...Response` or both are `to...Response`.